--- mod_ssl-2.8.26-1.3.35/configure~	2004-04-01 09:37:37.000000000 +0000
+++ mod_ssl-2.8.26-1.3.35/configure	2004-04-01 09:43:04.000000000 +0000
@@ -283,8 +283,7 @@
 
     #   determine Apache parameters
     apache=$APXS_PREFIX
-    APV=`$APXS_SBINDIR/$APXS_TARGET -v | grep "Server version:" |\
-         sed -e 's;^.*Apache/;;' -e 's; .*$;;'`
+    APV="@APACHE_VERSION@"
 else
     if [ ! -f "$apache/src/include/httpd.h" ]; then
         echo "$0:${T_MD}Error${T_ME}: Cannot find Apache 1.3 source tree under $apache" 1>&2
